src/connection/index.ts

```diff
@@ -34,8 +35,7 @@ export interface Connection {
   connector: Connector
   hooks: Web3ReactHooks
   type: ConnectionType
-  // TODO(WEB-3130): add darkmode check for icons
-  getIcon?(): string
+  getIcon?(isDarkMode: boolean): string
```

Comment:
Can you extend the tests for this file to include dark mode coverage?

Reply:
done